### PR TITLE
Rename setters/getters/builders for Dns classes to meet proto conventions

### PR DESCRIPTION
**Pull request by mziccard**

### CODE REVIEW

File: google-cloud-dns/src/main/java/com/google/cloud/dns/ChangeRequest.java [hunk 1]
```diff
@@ -137,13 +149,27 @@ public ChangeRequest build() {
    * Returns the name of the {@link Zone} associated with this change request.
    */
   public String zone() {
```
Comment (lesv) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ChangeRequest.java]: Should this be deprecated?

File: google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java [hunk 2]
```diff
@@ -79,45 +79,96 @@
...
+    @Deprecated
     public int rrsetsPerZone() {
+      return getRrsetsPerZone();
+    }
+
+    /**
+     * Returns the maximum allowed number of {@link RecordSet}s per {@link ZoneInfo} in the
+     * project.
+     */
+    public int getRrsetsPerZone() {
```
Comment (lesv) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java]: I hate suggesting cobol like names, but `Rrsets` doesn't strike me as helpful.  Why not `getResourceRecordSets` ?

File: google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java [hunk 3]
```diff
@@ -79,45 +79,96 @@
...
      */
+    @Deprecated
     public int totalRrdataSizePerChange() {
+      return getTotalRrdataSizePerChange();
+    }
+
+    /**
+     * Returns the maximum allowed size for total records in one ChangesRequest in bytes.
+     */
+    public int getTotalRrdataSizePerChange() {
```
Comment (lesv) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java]: TotalRrdata ??

File: google-cloud-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java [hunk 4]
```diff
@@ -401,16 +403,16 @@ public String params() {
...
-      System.out.printf("\tZones: %d%n", quota.zones());
-      System.out.printf("\tRecord sets per zone: %d%n", quota.rrsetsPerZone());
+      System.out.printf("\tZones: %d%n", quota.getZones());
+      System.out.printf("\tRecord sets per zone: %d%n", quota.getRrsetsPerZone());
       System.out.printf("\tRecord sets per DNS record: %d%n",
-          quota.resourceRecordsPerRrset());
-      System.out.printf("\tAdditions per change: %d%n", quota.rrsetAdditionsPerChange());
-      System.out.printf("\tDeletions per change: %d%n", quota.rrsetDeletionsPerChange());
+          quota.getResourceRecordsPerRrset());
+      System.out.printf("\tAdditions per change: %d%n", quota.getRrsetAdditionsPerChange());
```
Comment (lesv) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java]: `Rrset` vs. `ResourceRecords`

File: google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java [hunk 5]
```diff
@@ -79,45 +79,96 @@
...
+    @Deprecated
     public int rrsetAdditionsPerChange() {
+      return getRrsetAdditionsPerChange();
+    }
+
+    /**
+     * Returns the maximum allowed number of {@link RecordSet}s to add per {@link
+     * ChangeRequest}.
+     */
+    public int getRrsetAdditionsPerChange() {
```
Comment (lesv) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java]: `getResourceRecordSetAdditionsPerChange`

Comment (mziccard) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java]: While I kind of agree that rrset is unpleasant, naming was agreed with people from the DNS team so I believe we should just stick to it.

Comment (lesv) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java]: SGTM

File: google-cloud-dns/src/main/java/com/google/cloud/dns/ChangeRequest.java (hunk 1 above)
Comment (lesv) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ChangeRequest.java]: PING - did you get this one?

Comment (mziccard) [inline google-cloud-dns/src/main/java/com/google/cloud/dns/ChangeRequest.java]: Argh, sorry I missed it. I'll fix it in a separate PR.

**Pull request merged**